sentry_sdk/integrations/aws_lambda.py

```diff
@@ -297,10 +297,16 @@ def get_lambda_bootstrap():
     if "bootstrap" in sys.modules:
         return sys.modules["bootstrap"]
     elif "__main__" in sys.modules:
-        if hasattr(sys.modules["__main__"], "bootstrap"):
+        module = sys.modules["__main__"]
+        if hasattr(module, "awslambdaricmain") and hasattr(
```

Comment:
Can we add a comment to note this case is specific to Python 3.9? Also providing a link to docs/code where we can see this change will help a lot.

Reply:
I added some comments.

I can't find any documentation - I suspect the original author repeatedly ran a lambda which logged information about the contents of `sys.modules['__main__']` until they found the location of `bootstrap`